### PR TITLE
osd: admin cmd to dump pools statfs for a specific OSD

### PR DESCRIPTION
**Pull request by ifed01**
Signed-off-by: Igor Fedotov <ifedotov@suse.com>

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

File: src/osd/OSD.cc [hunk 1]
```diff
@@ -3206,6 +3206,30 @@ will start to track new ops received afterwards.";
...
+      ss << "Error dumping pool statfs: no poolid provided";
+      ret = -EINVAL;
+      goto out;
+    }
+
+    store_statfs_t st;
+    bool per_pool_omap_stats = false;
+
+    ret = store->pool_statfs(p, &st, &per_pool_omap_stats);
+    if (r < 0) {
```
Comment (tchaikov) [inline src/osd/OSD.cc:3223]:
```
../src/osd/OSD.cc:3223:9: error: 'r' was not declared in this scope
 3223 |     if (r < 0) {
      |         ^
```

**Comment by ifed01** [comment]
jenkins test make check

**Comment by ifed01** [comment]
jenkins test make check

**Comment by ifed01** [comment]
jenkins test make check

File: src/osd/OSD.cc (hunk 1 above)
Comment (ifed01) [reply src/osd/OSD.cc]: > ```
> ../src/osd/OSD.cc:3223:9: error: 'r' was not declared in this scope
>  3223 |     if (r < 0) {
>       |         ^
> ```

@tchaikov - thanks, fixed! Mind reviewing?

**Comment by tchaikov** [comment]
- https://pulpito.ceph.com/kchai-2021-04-30_03:09:53-rados-wip-kefu-testing-2021-04-30-0020-distro-basic-smithi/
- https://pulpito.ceph.com/kchai-2021-04-30_10:06:45-rados-wip-kefu-testing-2021-04-30-1335-distro-basic-smithi/

**Review by tchaikov — APPROVED** [review]

**Pull request merged**